packages/@webex/plugin-meetings/src/media/internal-media-core-wrapper.ts

```diff
@@ -0,0 +1,9 @@
+import {MediaConnection as MC} from '@webex/internal-media-core';
+
+/* We have this wrapper just because otherwise it's impossible to mock
+ * RoapMediaConnection and MultistreamRoapMediaConnection constructors in unit tests,
+ * because they are exported by @webex/internal-media-core as non-writable, non-configurable
+ * properties of MediaConnection and sinon doesn't allow spying on them or stubbing them.
+ */
+export const RoapMediaConnection = MC.RoapMediaConnection;
```

Comment:
Makes sense thanks for the comment , Jest would be the option but will take more time